students/mihwa-yang/users/views.py

```diff
@@ -1 +1,35 @@
-# Create your views here.
+import json
+
+from django.http  import JsonResponse
+from django.views import View
+from django.db    import IntegrityError
+
+from users.models import User
+
+class SignUpView(View): 
+    def post(self, request):
+        data  = json.loads(request.body)
+
+        if 'password' not in data.keys() or 'email' not in data.keys():
+            return JsonResponse({"MESSAGE": "KEY ERROR"}, status=400)
+
+        if data['email'].count('@') == 0 or data['email'].count('.') == 0:
+            return JsonResponse({"MESSAGE": "EMAIL FORM ERROR"}, status=400)
+        
+        PASSWORD_LENGTH = 8 
+        if len(data['password']) < PASSWORD_LENGTH:
+            return JsonResponse({"MESSAGE": "PASSWORD ERROR"}, status=400)
+
+        if User.objects.filter(email=data.get('email')).exists() == True \
+            or User.objects.filter(nickname=data.get('nickname')).exists() == True \
+            or User.objects.filter(phone=data.get('phone')).exists() == True:
+            return JsonResponse({"MESSAGE": "ALREADY EXIT ERROR"}, status=400)
```

Comment:
```suggestion
        if User.objects.filter(email=data.get('email')).exists()  \
            or User.objects.filter(nickname=data.get('nickname')).exists() \
            or User.objects.filter(phone=data.get('phone')).exists():
            return JsonResponse({"MESSAGE": "ALREADY EXIT ERROR"}, status=400)
```
.exits()의 반환값이 이미 boolean이기 때문에 == 으로 True/False을 구분하지 않으셔도 됩니다.(이렇게 짜는 것이 파이썬스럽다 라고 할 수 있습니다.)

Reply:
넵! 수정했습니다 👍